### ISSUE TITLE
Provide a standalone version of cascalog

### ISSUE THREAD

**Issue by BertrandDechoux** [nathanmarz/cascalog#84]
The last tutorial http://nathanmarz.com/blog/news-feed-in-38-lines-of-code-using-cascalog.html
was interesting but is getting old (clojure/hadoop and cascalog version).

For future version of Cascalog, it would be great to have a default standalone archive (a jar) that wouldn't require the user to build its own jar. The use case is that a user might just want to interact with its dataset using Cascalog regardless of the use (or not use) of Cascalog for the real requests.

It would be really nice to have it for Cascalog 2.0

Bertrand

**Comment by sritchie** [comment]
Hey @BertrandDechoux, I agree - did you ever make any progress on this? Would love to hear your thoughts.

**Comment by BertrandDechoux** [comment]
No. But I was in love with how Incanter made it easy : https://github.com/incanter/incanter/wiki#getstarted

I haven't looked at how it was done nor if it was easy for cascalog do to the same but that could be a start.
